code/game/machinery/computer/communications.dm

```diff
@@ -446,8 +446,8 @@
 			to_chat(user, "Central Command does not currently have a shuttle available in your sector. Please try again later.")
 			return
 
-		if(GLOB.sent_strike_team == 1)
-			to_chat(user, "Central Command will not allow the shuttle to be called. Consider all contracts terminated.")
+		if(GLOB.strike_team_sent == 1)
+			to_chat(user, "Central Command will not allow the shuttle to be called. Please stand by.</span>") //This may show up before Epsilon Alert/Before DS arrives
```

Comment:
```suggestion
		if(GLOB.strike_team_sent)
			to_chat(user, "Central Command does not allow the shuttle to be called at this time. Please stand by.</span>") //This may show up before Epsilon Alert/Before DS arrives
```